sql/create_train_val.go

```diff
@@ -91,11 +90,11 @@ func createMaxcomputeTable(target, origin string, db *DB, cond string) error {
 }
 
 func namingTrainAndValDataset() *trainAndValDataset {
-	uniq := time.Now().UnixNano() / 1e3
+	suffix := "sqlflow"
```

Comment:
@Yancey1989 why removing the random name? Shouldn't training and validation tables have a temporary name?

Reply:
Also, this PR is failing on MaxCompute unit tests. https://travis-ci.com/sql-machine-learning/sqlflow/jobs/224573082#L4551

Reply:
In one case, users need to **Inference on the whole validation data **, so the Inference Job need to know the validation table name, instead of a hidden random name. But I think we can name the table name to `<original_table_name>_tv_sqlflow, <original>_train_sqlflow, <orignal>_validatoin_sqlflow`.